diff -urN m17n-contrib-1.1.10-old/im/as-inscript.mim m17n-contrib-1.1.10/im/as-inscript.mim
--- m17n-contrib-1.1.10-old/im/as-inscript.mim	2011-03-31 10:16:58.336265533 +0530
+++ m17n-contrib-1.1.10/im/as-inscript.mim	2011-03-31 10:25:48.830681231 +0530
@@ -112,6 +112,7 @@
   ("2" "২")
   ("3" "৩")
   ("4" "৪")
+  ((G-4) "₹")
   ("5" "৫")
   ("6" "৬")
   ("7" "৭")
diff -urN m17n-contrib-1.1.10-old/im/bn-inscript.mim m17n-contrib-1.1.10/im/bn-inscript.mim
--- m17n-contrib-1.1.10-old/im/bn-inscript.mim	2010-04-14 10:38:02.000000000 +0530
+++ m17n-contrib-1.1.10/im/bn-inscript.mim	2011-03-31 10:25:48.826681243 +0530
@@ -75,6 +75,7 @@
   ("2" ?২)
   ("3" ?৩)
   ("4" ?৪)
+  ((G-4) "₹")
   ("5" ?৫)
   ("6" ?৬)
   ("7" ?৭)
diff -urN m17n-contrib-1.1.10-old/im/gu-inscript.mim m17n-contrib-1.1.10/im/gu-inscript.mim
--- m17n-contrib-1.1.10-old/im/gu-inscript.mim	2010-04-14 10:38:02.000000000 +0530
+++ m17n-contrib-1.1.10/im/gu-inscript.mim	2011-03-31 10:25:48.828681238 +0530
@@ -76,6 +76,7 @@
   ("3" ?૩)
   ("$" ?ૄ)
   ("4" ?૪)
+  ((G-4) "₹")
   ("5" ?૫)
   ("6" ?૬)
   ("7" ?૭)
diff -urN m17n-contrib-1.1.10-old/im/hi-inscript.mim m17n-contrib-1.1.10/im/hi-inscript.mim
--- m17n-contrib-1.1.10-old/im/hi-inscript.mim	2010-04-14 10:38:02.000000000 +0530
+++ m17n-contrib-1.1.10/im/hi-inscript.mim	2011-03-31 10:25:48.825681246 +0530
@@ -75,6 +75,7 @@
   ("3" ?३)
   ("$" ?$)
   ("4" ?४)
+  ((G-4) "₹")
   ("5" ?५)
   ("6" ?६)
   ("7" ?७)
diff -urN m17n-contrib-1.1.10-old/im/kn-inscript.mim m17n-contrib-1.1.10/im/kn-inscript.mim
--- m17n-contrib-1.1.10-old/im/kn-inscript.mim	2011-03-31 10:16:58.341265520 +0530
+++ m17n-contrib-1.1.10/im/kn-inscript.mim	2011-03-31 10:25:48.829681234 +0530
@@ -95,6 +95,7 @@
   ("3" ?೩)
   ("$" ?$)
   ("4" ?೪)
+  ((G-4) "₹")
   ("5" ?೫)
   ("6" ?೬)
   ("7" ?೭)
diff -urN m17n-contrib-1.1.10-old/im/ml-inscript.mim m17n-contrib-1.1.10/im/ml-inscript.mim
--- m17n-contrib-1.1.10-old/im/ml-inscript.mim	2011-03-31 10:16:58.339265525 +0530
+++ m17n-contrib-1.1.10/im/ml-inscript.mim	2011-03-31 10:25:48.824681249 +0530
@@ -175,6 +175,7 @@
   ("3" "൩")
   ("$" "$")
   ("4" "൪")
+  ((G-4) "₹")
   ("%" "%")
   ("5" "൫")
   ("^" "^")
diff -urN m17n-contrib-1.1.10-old/im/mr-inscript.mim m17n-contrib-1.1.10/im/mr-inscript.mim
--- m17n-contrib-1.1.10-old/im/mr-inscript.mim	2010-04-14 10:38:02.000000000 +0530
+++ m17n-contrib-1.1.10/im/mr-inscript.mim	2011-03-31 10:25:48.823681252 +0530
@@ -79,6 +79,7 @@
   ("3" ?३)
   ("$" ?$)
   ("4" ?४)
+  ((G-4) "₹")
   ("5" ?५)
   ("6" ?६)
   ("7" ?७)
diff -urN m17n-contrib-1.1.10-old/im/or-inscript.mim m17n-contrib-1.1.10/im/or-inscript.mim
--- m17n-contrib-1.1.10-old/im/or-inscript.mim	2011-03-31 10:16:58.344265508 +0530
+++ m17n-contrib-1.1.10/im/or-inscript.mim	2011-03-31 10:23:29.852096131 +0530
@@ -74,6 +74,7 @@
   ("3" "୩")
   ("$" "ର୍")
   ("4" "୪")
+  ((G-4) "₹")
   ("5" "୫")
   ("6" "୬")
   ("7" "୭")
diff -urN m17n-contrib-1.1.10-old/im/pa-inscript.mim m17n-contrib-1.1.10/im/pa-inscript.mim
--- m17n-contrib-1.1.10-old/im/pa-inscript.mim	2010-04-14 10:38:02.000000000 +0530
+++ m17n-contrib-1.1.10/im/pa-inscript.mim	2011-03-31 10:26:25.150567798 +0530
@@ -76,6 +76,7 @@
   ("2" ?੨)
   ("3" ?੩)
   ("4" ?੪)
+  ((G-4) "₹")
   ("5" ?੫)
   ("6" ?੬)
   ("7" ?੭)
diff -urN m17n-contrib-1.1.10-old/im/sd-inscript.mim m17n-contrib-1.1.10/im/sd-inscript.mim
--- m17n-contrib-1.1.10-old/im/sd-inscript.mim	2010-04-14 10:38:02.000000000 +0530
+++ m17n-contrib-1.1.10/im/sd-inscript.mim	2011-03-31 10:26:25.149567802 +0530
@@ -74,6 +74,7 @@
   ("3" ?३)
   ("$" ?$)
   ("4" ?४)
+  ((G-4) "₹")
   ("5" ?५)
   ("6" ?६)
   ("7" ?७)
diff -urN m17n-contrib-1.1.10-old/im/ta-inscript.mim m17n-contrib-1.1.10/im/ta-inscript.mim
--- m17n-contrib-1.1.10-old/im/ta-inscript.mim	2010-04-14 10:38:02.000000000 +0530
+++ m17n-contrib-1.1.10/im/ta-inscript.mim	2011-03-31 10:23:29.849096143 +0530
@@ -38,6 +38,7 @@
   ("3" ?3)
   ("$" ?$)
   ("4" ?4)
+  ((G-4) "₹")
   ("%" ?%)
   ("5" ?5)
   ("^" ?^)
diff -urN m17n-contrib-1.1.10-old/im/te-inscript.mim m17n-contrib-1.1.10/im/te-inscript.mim
--- m17n-contrib-1.1.10-old/im/te-inscript.mim	2011-03-31 10:16:58.350265491 +0530
+++ m17n-contrib-1.1.10/im/te-inscript.mim	2011-03-31 10:23:29.842096163 +0530
@@ -74,6 +74,7 @@
   ("3" ?౩)
   ("$" ?$)
   ("4" ?౪)
+  ((G-4) "₹")
   ("5" ?౫)
   ("6" ?౬)
   ("7" ?౭)
