--- im/ta-tamil99.mim.old	2007-05-29 10:35:58.000000000 +0530
+++ im/ta-tamil99.mim	2010-06-14 12:42:45.000000000 +0530
@@ -1,5 +1,5 @@
 ;; ta-tamil99.mim -- Tamil input method for tamil99 layout
-;; Copyright (c) 2006 Red Hat, Inc. All rights reserved.
+;; Copyright (c) 2006, 2010 Red Hat, Inc. All rights reserved.
 
 ;; This file is part of the m17n contrib; a sub-part of the m17n
 ;; library.
@@ -102,6 +102,7 @@
   ("a" ?அ)
   ("s" ?இ)
   ("d" ?உ)
+  ("f" ?்)
   ("F" ?ஃ)
   ("g" ?எ)
   ("h" ?க)
@@ -786,6 +787,26 @@
   ("C" ?௵)
   ("V" ?௶)
   ("B" ?௷)
+
+  ("^^" "^")
+  ("^c" "©")
+  ("^." "•")
+
+  ("^7" "‘")
+  ("^8" "’")
+  ("^9" "“")
+  ("^0" "”")
+  ("^S" " ")
+
+  ("^q" "ா")
+  ("^s" "ி")
+  ("^w" "ீ")
+  ("^d" "ு")
+  ("^e"  "ூ")
+  ("^g" "ெ")
+  ("^t" "ே")
+  ("^r" "ை")
+
   ))
 
 (state
